modules/effects/src/effects_runner.ts

```diff
@@ -17,6 +19,7 @@ export class EffectsRunner implements OnDestroy {
     if (!this.effectsSubscription) {
       this.effectsSubscription = this.effectSources
         .toActions()
+        .pipe(tap(action => throwErrorOnInvalidAction(action)))
```

Comment:
We don't need to to verify the action here. The action returned will be dispatched to the store, and will be verified there.

Reply:
You are absolutely right. The circle closes at the place! 
`EffectsRunner.start() => Store.next() => ActionsSubject.next()` :)

Reply:
Its removed!